### ISSUE TITLE
Folder picker interface needs improvement

### ISSUE THREAD

**Issue by Dataweaver42** [krypto5863/COM3D2.ExtendedPresetManagement#1]
When the number of folders involved is small, the names are short, and there's little to no nesting, the existing folder picker interface works reasonably well. But once names start getting long and extensive nesting comes into play, it becomes increasingly awkward to use. 

Consider replacing the flat list of folder names with select buttons next to each with a single tree control and a single Select button underneath that switches you to whichever folder is highlighted in the tree control.

**Comment by krypto5863** [comment]
Yeah the interface does suck pretty hard. I'll work on it some day.
